supervisor/coresys.py

```diff
@@ -114,6 +110,33 @@ async def load_config(self) -> Self:
         await self.config.read_data()
         return self
 
+    async def init_websession(self) -> None:
+        """Initialize global aiohttp ClientSession."""
+        if self.core.state in VALID_API_STATES:
+            # Make sure we don't reinitialize the session if the API is running (see #5851)
+            raise RuntimeError(
+                "Initialize ClientSession is not safe when API is running"
+            )
+
+        if self._websession:
+            await self._websession.close()
+
+        resolver = aiohttp.AsyncResolver()
+
+        # pylint: disable=protected-access
+        _LOGGER.debug(
+            "Initialize ClientSession with AsyncResolver. Using nameservers %s",
```

Comment:
```suggestion
            "Initializing ClientSession with AsyncResolver. Using nameservers %s",
```
(not as bad as the previous one, but still)